src/lib/comp/LayoutSearchFilter.svelte

```diff
@@ -0,0 +1,87 @@
+<script lang="ts">
+	import Input from '$lib/controls/Input.svelte';
+	import TextArea from '$lib/controls/TextArea.svelte';
+
+	export let id: string;
+	export let label: string = "Find:";
+	export let items: { [name: string]: any };
+	export let filterFunc: (itemKey:string, text:string) => boolean;
+	export let clearSearchFunc: () => void = () => {};
+	export let searchText: string = '';
+	export let title: string = '';
+	export let textArea: boolean = false;
+	export let autoExpand: boolean = false;
+	export let numResults: number = 0;
+	export let showNoneForBlank: boolean = false;
+	export let searching: boolean = false;
```

Comment:
I don't think this field is read from. Can it be removed or is it going to be used in the future?

Reply:
Which one specifically? I see 4 lines here. These are all planned to be used.

Reply:
I was referring to line 16 or the searching field. But if that's going to be used in the future, it's fine.